ci_scripts/sync_target_database.py

```diff
@@ -80,6 +78,20 @@ def write_news_file(item_text: str, file_name: str) -> Path:
     return news_file_path
 
 
+def write_news_file_from_targets(added: MbedTargets, removed: MbedTargets, news_file_name: str) -> Path:
+    """Creates and writes a news file from the added and removed targets.
+
+    Args:
+        added: added MbedTargets
+        removed: removed MbedTargets
+        news_file_name: name of the news file
+    """
+    news_item_text_added = create_news_item_text("Targets added: ", added)
+    news_item_text_removed = create_news_item_text("Targets removed: ", removed)
+    news_item_text = f"{news_item_text_added} {news_item_text_removed}"
```

Comment:
Should there be some sort of separator between two messages? I'm not sure what'd work best. As it currently stands this would read: 
`New boards added: u-blox NINA-B1, Multitech xDOT New boards removed: NiceBoard1, HelloWorld-X3` - I can see it being hard to read the changelog.
How about: `Offline database update. Removed: u-blox NINA-B1, Multitech xDOT. Added: NiceBoard1, HelloWorld-X3`.

Reply:
I'll add a separator. I'm not sure using the term "boards" is the best thing here so I think I'll stick with the current wording, with separator.

Reply:
Oh, I was trying to mimic what the current message would be and failed. 😢 
For posterity: 
`Targets added:  u-blox NINA-B1, Multitech xDOT Targets removed: NiceBoard1, HelloWorld-X3` needs a separator as it's not readable.

Reply:
Added a full stop in 793e31c